concepts/observability-strategy/configurable-tracing/02-tracing-collectors.md

```diff
@@ -2,9 +2,32 @@
 
 ## Requirements
 
+The agent's responsibility is to collect all span data in the cluster (Istio, Serverless, Eventing, custom) and ship them in a configurable way to targets. The agent is not meant to be used as backend system, so ideally it is even stateless.
+
+- Collection
+  - Support OTLP push approach, ideally with service discovery. A lot of apps are based on the pull approach and should not be rewritten or operated with a sidecar only for transforming that approach.
+  - Support push-based model using OTLP.
+  - Scale-out must be possible dependent on incoming traffic
+- Filter
+  - Support filtering traces/spans per pipeline based on the following criteria:
+     - Source by Namespace or Pod name and Kubernetes labels
+  - Simple tail-based sampling based on percentage, potentially also by other criterias
```

Comment:
```suggestion
  - Simple tail-based sampling based on percentage, potentially also by other criteria
```

"criteria" is already [plural](https://www.merriam-webster.com/dictionary/criteria). If you mean one, it's "criterion".